Yui button

yuanyuli opened the pull request:
新加功能描述:
1 可以添加source属性设置图像
2 button状态更改可以通过动画方式改变

Commit:
功能增加 :
1 button 写入Source属性 可添加Image
2 设置ActiveStyle 可通过动画更改Style的更新

Commit:
Btton 添加Disabled 属性时 背景呈灰色

Commit:
增加animations属性

Commit:
修正安卓上不能显示动画的问题

ldn0x7dc commented:
Great job! 
以下是建议再修改的：
1. **animated** prop should default to false
2. pls use style/activeStyle, imageStyle/activeImageStyle, fontStyle/**activeFontStyle**(_not activeFontColor any more_)
3. pls make demo cover more cases and be more elegant
4. 请支持图片和文字水平摆放
5. pls use text prop instead of wrapped children

Commit:
Change
1. animated prop should default to false
2. pls use style/activeStyle, imageStyle/activeImageStyle, fontStyle/activeFontStyle(not activeFontColor any more)
3. 更新Demo展示
Add
1. 支持图片和文字水平摆放
2. 支持text&activeText,source&activeSource
